src/store/composesSlice.js

```diff
@@ -53,6 +53,7 @@ export const selectComposeById = (state, composeId) => {
       created_at: compose.created_at,
       id: compose.id,
       imageName: compose.image_name || compose.id,
+      ImageDescription: compose.image_description || compose.id,
```

Comment:
Let's unify the capitalization from ImageDescription -> imageDescription

Reply:
good catch, thank you